### PR TITLE
Apex smart contracts deployment

### PR DESCRIPTION
**Pull request by milos-ethernal**
# Description

Apex contracts deployment

CAUTION: do not merge this into `develop` branch. This will be part of blade-apex fork one day

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [ ] Hotfix (change that solves an urgent issue, and requires immediate attention)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [ ] I have added the relevant labels
- [ ] I have updated the official documentation
- [ ] I have added sufficient documentation in code

## Testing

- [ ] I have tested this code with the official test suite
- [x] I have tested this code manually

### Manual tests

Test deployed contracts with [this test](https://github.com/Ethernal-Tech/apex-bridge/blob/blade_test/eth/blade_test.go).

### CODE REVIEW

**Comment by milos-ethernal** [comment]
I have read the CLA Document and I hereby sign the CLA

File: consensus/polybft/contracts_initializer.go [hunk 1]
```diff
@@ -435,3 +435,194 @@ func isNativeRewardToken(cfg PolyBFTConfig) bool {
 func IsNativeStakeToken(stakeTokenAddr types.Address) bool {
 	return stakeTokenAddr == contracts.NativeERC20TokenContract
 }
+
+// initProxies initializes proxy contracts, that allow upgradeability of contracts implementation
```
Comment (igorcrevar) [inline consensus/polybft/contracts_initializer.go]: move all of these into `apex_contracts_initializer.go`

File: consensus/polybft/contractsapi/apex-artifacts-gen/main.go [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+		str += fmt.Sprintf("var %sArtifact string = `%s`\n", v.Name, dst.String())
+	}
+
+	output, err := format.Source([]byte(str))
+	if err != nil {
+		fmt.Println(str)
+		log.Fatal(err)
+	}
+
+	if err = common.SaveFileSafe(currentPath+"/../apex_sc_data.go", output, 0600); err != nil {
```
Comment (igorcrevar) [inline consensus/polybft/contractsapi/apex-artifacts-gen/main.go]: neatpick, but use `path.Join`, maybe one day we will support windows too :)

**Review by igorcrevar — APPROVED** [review]
```
maxNumberOfTransactions = 5
timeoutBlocksNumber     = 5
```
will be passed through polybft config in another task

**Pull request merged**